<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2DScale</class>
 <widget class="QDialog" name="Dialog2DScale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Interval Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupX">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>x-range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioAutoCalcX">
        <property name="text">
         <string>auto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioManualX">
        <property name="text">
         <string>manual adjustment:</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>from</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>to</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="DoubleSpinBox" name="doubleSpinMaxX">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="decimalsOption">
           <set>DoubleSpinBox::DecimalsAsMin|DoubleSpinBox::DecimalsByShortcuts|DoubleSpinBox::InsertDecimals</set>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="sizeHintPolicy">
           <enum>DoubleSpinBox::SizeHintByValue</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="DoubleSpinBox" name="doubleSpinMinX">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="decimalsOption">
           <set>DoubleSpinBox::DecimalsAsMin|DoubleSpinBox::DecimalsByShortcuts|DoubleSpinBox::InsertDecimals</set>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="sizeHintPolicy">
           <enum>DoubleSpinBox::SizeHintByValue</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupY">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>y-range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioAutoCalcY">
        <property name="text">
         <string>auto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioManualY">
        <property name="text">
         <string>manual adjustment:</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>from</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>to</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="DoubleSpinBox" name="doubleSpinMinY">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="decimalsOption">
           <set>DoubleSpinBox::DecimalsAsMin|DoubleSpinBox::DecimalsByShortcuts|DoubleSpinBox::InsertDecimals</set>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="sizeHintPolicy">
           <enum>DoubleSpinBox::SizeHintByValue</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="DoubleSpinBox" name="doubleSpinMaxY">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="decimalsOption">
           <set>DoubleSpinBox::DecimalsAsMin|DoubleSpinBox::DecimalsByShortcuts|DoubleSpinBox::InsertDecimals</set>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="sizeHintPolicy">
           <enum>DoubleSpinBox::SizeHintByValue</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupValue">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>values</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioAutoCalcValue">
        <property name="text">
         <string>auto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioManualValue">
        <property name="text">
         <string>manual adjustment:</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>from</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>to</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="DoubleSpinBox" name="doubleSpinMinValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="decimalsOption">
           <set>DoubleSpinBox::DecimalsAsMin|DoubleSpinBox::DecimalsByShortcuts|DoubleSpinBox::InsertDecimals</set>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="sizeHintPolicy">
           <enum>DoubleSpinBox::SizeHintByValue</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="DoubleSpinBox" name="doubleSpinMaxValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="decimalsOption">
           <set>DoubleSpinBox::DecimalsAsMin|DoubleSpinBox::DecimalsByShortcuts|DoubleSpinBox::InsertDecimals</set>
          </property>
          <property name="minimum">
           <double>-999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="sizeHintPolicy">
           <enum>DoubleSpinBox::SizeHintByValue</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Hint: Control the number of decimals in the spin boxes by pressing Ctrl+ or Ctrl-</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>doubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioAutoCalcX</tabstop>
  <tabstop>radioManualX</tabstop>
  <tabstop>doubleSpinMinX</tabstop>
  <tabstop>doubleSpinMaxX</tabstop>
  <tabstop>radioAutoCalcY</tabstop>
  <tabstop>radioManualY</tabstop>
  <tabstop>doubleSpinMinY</tabstop>
  <tabstop>doubleSpinMaxY</tabstop>
  <tabstop>radioAutoCalcValue</tabstop>
  <tabstop>radioManualValue</tabstop>
  <tabstop>doubleSpinMinValue</tabstop>
  <tabstop>doubleSpinMaxValue</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog2DScale</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog2DScale</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualValue</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinMinValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualY</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinMinY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualY</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinMaxY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualX</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinMinX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualX</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinMaxX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioManualValue</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinMaxValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
